docs/understanding-airbyte/basic-normalization.md

```diff
@@ -281,3 +281,12 @@ Note that all the choices made by Normalization as described in this documentati
 * to build a [custom SQL view](../operator-guides/transformation-and-normalization/transformations-with-sql.md) with your own naming conventions
 * to export, edit and run [custom dbt normalization](../operator-guides/transformation-and-normalization/transformations-with-dbt.md) yourself
 * or further, you can configure the use of a custom dbt project within Airbyte by following [this guide](../operator-guides/transformation-and-normalization/transformations-with-airbyte.md).
+
+## CHANGELOG
```

Comment:
Is it good @avaidyanatha if I'm adding this similar to connectors?

Reply:
@ChristopheDuong Absolutely, this is good!